lib/rules/block-opening-brace-newline-after/__tests__/index.js

```diff
@@ -432,3 +432,29 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: ['always', { ignore: ['rules'] }],
+	fix: true,
+	accept: [
+		{
+			code: 'a { color: pink; }',
+		},
+		{
+			code: 'a {\ncolor: pink; }',
+		},
+		{
+			code: '@media print {\na {\ncolor: pink; } }',
+		},
+	],
+	reject: [
+		{
+			code: '@media print { a {\ncolor: pink; } }',
+			fixed: '@media print {\n a {\ncolor: pink; } }',
```

Comment:
```suggestion
			code: '@media print { a { color: pink; } }',
			fixed: '@media print {\n a { color: pink; } }',
```

Let's check that only the at-rule is fixed.